docs/source/index.rst

```diff
@@ -80,6 +80,32 @@ python files. The recommended way to use this is with a context manager
     with adl.open('newfile', 'rb') as f:
         df = pd.read_csv(f) # read into pandas.
 
+To seamlessly handle remote path representations across all supported platforms,
+the main API will take in numerous path types: string, Path/PurePath, and
+AzureDLPath. On Windows in particular, you can pass in paths separated by either
+forward slashes or backslashes.
```

Comment:
For clarity, these are just the paths for the remote files, or also for local files for upload? For example, can a windows user put in the local file path /my/file/for/upload.txt? How would that be properly translated to a file on the local NTFS filesystem?